shared/test/unit/mocks/mock_navigator_moz_apps.js

```diff
@@ -41,6 +34,53 @@ var MockNavigatormozApps = {
     }
   },
 
+  mCreateDOMRequest: function() {
```

Comment:
Maybe a bit overcomplicated, don't have strong opinion on that, ideally it should be in a separate mock file, but to use it every test should explicitly ```require``` it :(